javascript/javascript3/week3/visual/main.js

```diff
@@ -0,0 +1,30 @@
+moveElement(document.querySelector("li"), { x: 20, y: 300 }).then(() => {
+  console.log("Element has been moved");
+});
+
+async function translateOneByOne() {
+  const redCircle = document.querySelector(".marks").firstElementChild;
+  await moveElement(redCircle, {
+    x: 20 - parseInt(redCircle.style.left),
```

Comment:
It looks like you tried to determine the position of the redCircle element by accessing its styles and calculating the position of the target programmatically. That's a really good idea! 👍 However, using the `style` property only returns a value if the style has actually been set. In this case it's simply `''` empty string. For this exercise it is totally okay to hardcode the x,y values, so to make it easier you could go with experimenting and setting the fitting absolute values for now.